packages/react-components/src/molecules/ToastCard.tsx

```diff
@@ -23,21 +21,52 @@ const iconStyles = css({
   paddingRight: `${15 / perRem}em`,
 });
 
+const emberStyles = css({
+  color: ember.rgb,
+  fill: ember.rgb,
+});
+
+const leadStyles = css({
+  color: lead.rgb,
+  fill: lead.rgb,
+});
+
+type Type = 'alert' | 'attachment' | 'live';
```

Comment:
Are we going to have multiple types and type mappings to icons?

Reply:
Type mapping to icon and colour scheme. Alert icons are red with red text.